src/Ledger/Transaction.lagda

```diff
@@ -192,4 +210,23 @@ the transaction body are:
   instance
     HasCoin-TxOut : HasCoin TxOut
     HasCoin-TxOut .getCoin = coin ∘ proj₁ ∘ proj₂
+
+  getDataHashes : ℙ TxOut → ℙ DataHash
```

Comment:
These two functions should be moved to `Ledger.Utxo` since they're only used for the `UTXO` rule.